diff -Naur assimp-6.0.4/code/AssetLib/IFC/IFCLoader.cpp assimp-6.0.4.new/code/AssetLib/IFC/IFCLoader.cpp
--- assimp-6.0.4/code/AssetLib/IFC/IFCLoader.cpp	2026-01-24 21:57:37.000000000 +0100
+++ assimp-6.0.4.new/code/AssetLib/IFC/IFCLoader.cpp	2026-02-25 11:31:01.581491844 +0100
@@ -53,7 +53,7 @@
 #ifdef ASSIMP_USE_HUNTER
 #include <minizip/unzip.h>
 #else
-#include <unzip.h>
+#include <minizip/unzip.h>
 #endif
 #endif
 
diff -Naur assimp-6.0.4/code/Common/ZipArchiveIOSystem.cpp assimp-6.0.4.new/code/Common/ZipArchiveIOSystem.cpp
--- assimp-6.0.4/code/Common/ZipArchiveIOSystem.cpp	2026-01-24 21:57:37.000000000 +0100
+++ assimp-6.0.4.new/code/Common/ZipArchiveIOSystem.cpp	2026-02-25 11:26:42.093307557 +0100
@@ -54,7 +54,7 @@
 #ifdef ASSIMP_USE_HUNTER
 #    include <minizip/unzip.h>
 #else
-#    include <unzip.h>
+#    include <minizip/unzip.h>
 #endif
 
 namespace Assimp {
